deployment/v2/1_createGenesis.ts

```diff
@@ -90,15 +90,18 @@ async function main() {
 
     // Check if it's mainnet deployment
     const isMainnet = deployParameters["isMainnet"] === true ? true : false;
+    let isSovereignChain = deployParameters["isSovereignChain"] === true ? true : false;
 
     if (isMainnet === true) {
+        // If is mainnet, can't be sovereign
```

Comment:
the flag is mainnet is regarding to the address used on mainnet, it can be sovereign of course! ^^